File: docs/source/_snippets/simulation/environment_setup/create_bodies_1.py

# define bodies in simulation
bodies_to_create = ["Sun", "Earth", "Moon"]

# create body settings dictionary
body_settings = environment_setup.get_default_body_settings(
        bodies_to_create, "SSB", "J2000")

# create body system
bodies = environment_setup.create_system_of_bodies(body_settings)

File: docs/source/_snippets/simulation/environment_setup/create_bodies_1.cpp
// load spice kernels
spice_interface::load_standard_spice_kernels()

// define bodies in simulation
std::vector<std::string> bodiesToCreate = {"Sun", "Earth", "Moon"};

// create body settings map
std::map<std::string, std::shared_ptr<BodySettings>> bodySettings =
    getDefaultBodySettings(bodiesToCreate);

// create body system object
NamedBodyMap bodyMap = createBodies(bodySettings);
